examples/todolist/src/index.ts

```diff
@@ -0,0 +1,163 @@
+import { promises as fsPromises } from 'fs';
+import { createServer } from 'http';
+import { join } from 'path';
+import { createRouter, FromSchema, Response, useOpenAPI } from '@whatwg-node/router';
+
+const TodoSchema = {
+  type: 'object',
+  properties: {
+    id: { type: 'string' },
+    content: { type: 'string' },
+  },
+  required: ['id', 'content'],
+  additionalProperties: false,
+} as const;
+
+type Todo = FromSchema<typeof TodoSchema>;
+
+const todos: Todo[] = [];
+
+export const router = createRouter({
+  plugins: [
+    useOpenAPI({
+      baseOas: {
+        openapi: '3.0.1',
+        info: {
+          title: 'Todo List Example',
+          description: 'A simple todo list example with @whatwg-node/router',
+          version: '1.0.0',
+        },
+        components: {},
+      },
+    }),
+  ],
+})
+  .addRoute({
```

Comment:
Maybe we can move this `addRoute` inside the constructor to avoid mistakes by exporting `router` early.